services/src/main/java/org/exoplatform/addons/gamification/service/configuration/ChallengeServiceImpl.java

```diff
@@ -95,8 +96,8 @@ public Challenge updateChallenge(Challenge challenge, String username) throws Il
   }
 
   @Override
-  public boolean canAddChallenge(String currentUser) throws Exception {
-    return spaceService.getManagerSpaces(currentUser).getSize() > 0;
+  public boolean canAddChallenge(String currentUser)  {
+    return ConversationState.getCurrent().getIdentity().isMemberOf("/platform/administrators");
```

Comment:
This should be configured with an external configuration in the InitParams of the service and possibly filled by a value from exo.properties.

Reply:
done!